elasticsearch/backup.tf

```diff
@@ -35,9 +35,11 @@ resource "google_storage_bucket" "es-snapshot-bucket" {
 }
 
 # Grant sa permission to write to bucket
-resource "google_storage_bucket_iam_binding" "es-sa-binding" {
+resource "google_storage_bucket_iam_binding" "es-sa-binding-edit" {
```

Comment:
why the re-name?

Reply:
oh, I think I may have typed something into the wrong active window.